pkg/apis/cr/v1alpha1/types.go

```diff
@@ -51,8 +51,10 @@ var _ runtime.Object = (*ActionSet)(nil)
 type ActionSet struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata"`
-	Spec              *ActionSetSpec   `json:"spec,omitempty"`
-	Status            *ActionSetStatus `json:"status,omitempty"`
+	// Spec defines the specification for ActionSet
```

Comment:
```suggestion
	// Spec defines the specification for ActionSet, for example which action should be performed and with which blueprint etc. It also has details about the profile and any other object that might be used by referred blueprint.
```

Reply:
feel free to make these sentences better with the help of grammar checker.

Reply:
Alright!